share/completions/minikube.fish

```diff
@@ -0,0 +1,79 @@
+function __minikube_no_command
+    set -l cmd (commandline -poc)
+    set -e cmd[1]
+    if test (count $cmd) -eq 0
+        return 0
+    end
+    return 1
+end
+
+function __minikube_using_command
+    set cmd (commandline -poc)
+    if test (count $cmd) -gt 1
+        if test $argv[1] = $cmd[2]
+            return 0
+        end
+    end
+    return 1
+end
+
+complete -c minikube -f -n "__minikube_no_command" -a addons -d "Modify minikube's kubernetes addons"
+complete -c minikube -f -n "__minikube_no_command" -a completion -d "Outputs minikube shell completion for the given shell (bash)"
```

Comment:
You specify it as `minikube completion SHELL`, however they only support bash it seems. Perhaps modify the description to indicate that it is given as an argument to `completion`, or skip it completely, not sure how valuable it is for within fish to generate completions for bash, but it has potential use cases.